PhysicsTools/PatAlgos/plugins/PATTracksToPackedCandidates.cc

```diff
@@ -0,0 +1,203 @@
+//Based on: PhysicsTools/PatAlgos/plugins/PATLostTracks.cc
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/RecoCandidate/interface/RecoChargedCandidate.h"
```

Comment:
```suggestion
```
seems not necessary

Reply:
This comment is included in this commit: https://github.com/cms-sw/cmssw/pull/30313/commits/6e7159380e61d7f6bf7342a1f71162fbcace502a